src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1922,4 +1922,7 @@ Additional information: {1}</value>
   <data name="External_sources" xml:space="preserve">
     <value>External Sources</value>
   </data>
+  <data name="Fix_analyzer_warnings_and_errors" xml:space="preserve">
+    <value>Fix analyzer warnings and errors</value>
```

Comment:
Would be good to get a screenshot of the code cleanup screen with this, as I can't quite convince myself this makes sense in context. Seems like it would need some qualifier to distinguish from built in stuff.

Reply:
I am of two minds here. If you know how the IDE/compiler works, then you think of everything as an "analyzer". So, I can see this wording being confusing. Yet the built in IDE functionality being analyzers in an implementation detail and I do not think most users are aware of this.

I chose "Fix analyzer warnings and errors" because I expect that most users will interpret analyzer as "Things that appear in my analyzers node under my project"

<img width="255" alt="image" src="https://user-images.githubusercontent.com/9797472/155630663-de5292d0-7af5-47be-a6de-72ec62b93ff6.png">

But I am not super attached to this wording. Perhaps something more descriptive like one of these:

1. "Fix all analyzer warnings and errors"
1. "Fix additional analyzer warnings and errors"
1. "Fix warnings and errors from third-party analyzers"
1. "Fix warnings and errors from referenced analyzers"

Reply:
Let me know which one sounds the most correct @davidwengier

Reply:
You make a good point, maybe the IDE diagnostics aren't seen as analyzers. I'm happy to leave this and see if there is feedback, but I also really like "referenced analyzers". Up to you :)

Reply:
I think I would like to pitch in ``Fix all warnings and errors (including referenced analyzers)``.

Reply:
i def think we should mention 3rd party somehow.

Reply:
I think we should mention EditorConfig as well. Can we have 2 options? 1. `Fix analyzer warnings and errors set in EditorConfig`. 2. `Fix all warnings and errors set in EditorConfig` (the second option would read all settings in EditorConfig including code style, whitespace, and analyzers).

Reply:
alright, changed the wording and added the option you suggested @mikadumont